src/otaclient/ota_core.py

```diff
@@ -984,7 +983,8 @@ def execute(self) -> None:
         """
         try:
             self._execute_update()
-            shutil.rmtree(self._resource_dir_on_standby, ignore_errors=True)
+            # NOTE(20250818): preserve the OTA resource dir to speed up next OTA
+            # shutil.rmtree(self._resource_dir_on_standby, ignore_errors=True)
```

Comment:
I think we can delete this `rmtree` line :)

Reply:
Updated in [324b952](https://github.com/tier4/ota-client/pull/640/commits/324b9528374851e8fac09d3cd4cdc4d15e28a3ef)